hikari/impl/event_manager_base.py

```diff
@@ -351,8 +349,8 @@ def get_listeners(
 
     def unsubscribe(
         self,
-        event_type: typing.Type[event_manager_.EventT_co],
-        callback: event_manager_.CallbackT[event_manager_.EventT_co],
+        event_type: typing.Type[typing.Any],
```

Comment:
These shouldn't be typed as Any

Reply:
They have been for a while, just in bot and not in the event manager. Explanation here: https://github.com/hikari-py/hikari/pull/945#issuecomment-1000555490

Reply:
I am well aware of that, this just hasn't ever been an issue for anybody in this context and I don't think progressively spreading shit type hints to places where it hasn't mattered to anybody to support a bug in a specific type checker is really a good thing to be doing.